src/edit/DistortableImage.EditToolbar.js

```diff
@@ -130,53 +128,70 @@ var EditOverlayAction = LeafletToolbar.ToolbarAction.extend({
 	}),
 
   EnableEXIF = EditOverlayAction.extend({
+		options: {
+			toolbarIcon: {
+				html: '<span class="fa fa-compass"></span>',
+				tooltip: "Enable EXIF",
+				title: "Geocode Image"
+			}
+		},
+
+		addHooks: function() {
+			var image = this._overlay._image;
+			EXIF.getData(image, L.EXIF(image));
+		}
+	}),
+	
+	Restore = EditOverlayAction.extend({
+		options: {
+			toolbarIcon: {
+				html: '<span class="fa fa-fast-backward"></span>',
+				tooltip: 'Restore to initial proportions',
+				title: 'Restore'
```

Comment:
I actually had suggested earlier about removing the `title` field from here as it seems redundant, [refer this](https://github.com/Leaflet/Leaflet.toolbar/wiki/API-Reference#toolbariconoptions). What do you think?

Reply:
@rexagod yeah title isn't doing anything here very valid

I'm going to send a commit to update this one - but the rest are great FTO material.. do you want to open one ☺️

Reply:
nevermind I have another PR open where im fixing it because I'm updating our icons